index.html

```diff
@@ -102,7 +102,7 @@ <h2 id="project_tagline">explore open source projects and <a href="http://nikcod
         <ol>
           <li>Tag bugs and feature requests that would be a good place to start</li>
           <li>Grab the URL to this list of tasks so that others can get to it easily</li>
-          <li><a href="https://github.com/login">Sign in to GitHub</a> and <a href="https://github.com/dahlbyk/up-for-grabs.net/edit/gh-pages/_data/projects.yml" title="Edit projects.yml">add the project to this list</a></li>
+          <li><a href="https://github.com/login">Sign in to GitHub</a> and <a href="https://github.com/dahlbyk/up-for-grabs.net/blob/gh-pages/README.md">check out the README.</a></li>
```

Comment:
Since the readme is small, maybe link to the project page instead of the blob?

https://github.com/dahlbyk/up-for-grabs.net#testing-the-site-locally